pytradfri/mood.py

```diff
@@ -1,22 +1,23 @@
 """Represent a mood on the gateway."""
+from __future__ import annotations
+
 from .const import ROOT_MOODS
-from .resource import ApiResource
+from .resource import ApiResource, TypeRaw
 
 
 class Mood(ApiResource):
     """Mood."""
 
-    def __init__(self, raw, parent):
+    def __init__(self, raw: TypeRaw, parent: str) -> None:
         """Create object of class."""
         super().__init__(raw)
         self._parent = parent
 
     @property
-    def path(self):
+    def path(self) -> list[str]:
         """Path."""
-        return [ROOT_MOODS, self._parent, self.id]
+        return [ROOT_MOODS, str(self._parent), str(self.id)]
```

Comment:
Yeah, the parent should be a string already according to our typing. We need to check that if we think it's not true.

Reply:
Odd, it looks like strings here:
https://github.com/home-assistant-libs/pytradfri/blob/2fc8fa0ec1a639a0ace6093b1311d2009d453123/pytradfri/gateway.py#L212

Reply:
Oh, I believe moods linked to a group is of type `int` and not `str`. Let me push that code and we can have a look at both at the same time.

Reply:
We didn't resolve this, I think.

Reply:
Sorry, I misunderstood you in the other PR. This is what I meant by converting to str and the looking into it later. Should we revert PR?